pywbem/cim_obj.py

```diff
@@ -2511,6 +2511,72 @@ def to_wbem_uri(self, format='standard'):
 
         return _ensure_unicode(''.join(ret))
 
+    @staticmethod
+    def from_instance(class_, instance, namespace=None, host=None,
+                      strict=False):
```

Comment:
A change log entry is missing for this new method.

Reply:
I added an issue for the new method as documentation since it is a general enhancement and added that to title of the pr.

Done